docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+bar
```

Comment:
`Giuseppe’s bar` should be better, assuming it is his name and not some random placeholder string.